collection-pipeline/pom.xml

```diff
@@ -57,6 +57,14 @@
           </execution>
         </executions>
       </plugin>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+                <source>16</source>
+                <target>16</target>
+            </configuration>
+        </plugin>
```

Comment:
This is now set in the parent pom.xml. Please rebase your changes on top of `master`. Let's see if we get a passing build.

Reply:
You have the same configuration elsewhere too. Please check throughout the code.